pkg/app/instances/docker.go

```diff
@@ -104,6 +127,19 @@ func (m *DockerInstanceManager) CreateHost(zone string, _ *apiv1.CreateHostReque
 	if err != nil {
 		return nil, fmt.Errorf("failed to start docker container: %w", err)
 	}
+	execConfig := container.ExecOptions{
+		Cmd:          []string{"chown", "httpcvd:httpcvd", uaMountTarget},
+		AttachStdout: false,
+		AttachStderr: false,
+		Tty:          false,
+	}
+	execRes, err := m.Client.ContainerExecCreate(ctx, createRes.ID, execConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to change mode of user_artifacts diferctory: %w", err)
```

Comment:
rephrase error message, it's the same error message as below, we need to be able to determine which line failed.